recipe/meta.yaml

```diff
@@ -302,8 +303,8 @@ outputs:
         - python
         - {{ pin_subpackage('ray-default', exact=True) }}
         - pandas
+        - pyarrow >=6.0.1
```

Comment:
```suggestion
        - pyarrow >=6.0.1,<7.0.0
```
here and elsewhere

Reply:
We actually don't have pyarrow 6. We have 4, 8, 10 and 11.

Reply:
is that going to be a problem at runtime? Or can we get away with not having to build pyarrow 6

Reply:
I think it should be ok. Anyway introducing `>=6.0.1,<7.0.0` would probably cause conflicts all over the place because pyarrow depends on arrow-cpp, which has `pin_subpackage("arrow-cpp", max_pin="x.x.x")` as a run_exports.